tests/api_test/api_test.cpp

```diff
@@ -597,15 +597,15 @@ bind_tailcall_test(_In_ struct bpf_object* object)
 
 #define SOCKET_TEST_PORT 0x3bbf
 
-void
-send_traffic(IPPROTO protocol, bool is_ipv6)
+void send_traffic(IPPROTO protocol, bool is_ipv6)
 {
     const char* message = CLIENT_MESSAGE;
     sockaddr_storage destination_address{};
     PSOCKADDR local_address = nullptr;
     int local_address_length = 0;
 
-    if (protocol == IPPROTO_UDP) {
+    if (protocol == IPPROTO_UDP)
+    {
```

Comment:
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#formatting-notes says
> Our coding conventions follow the [LLVM coding standards](https://llvm.org/docs/CodingStandards.html) with the following over-rides:

>
> * Source lines MUST NOT exceed 120 columns.
> * Single-line if/else/loop blocks MUST be enclosed in braces.

The LLVM link above takes you to a page that shows that `{` goes on end of line not on own line.  So please revert these.

Reply:
The .\format-code.ps1  updates all the files for white spaces etc. I am skipping that for now to avoid changes in other files

Reply:
format-code.ps1 shouldn't be adding a new line between `if` and `{`.  if it is, please file an issue to track.